simpledotorg/simple-server
# Monthly state report download

**Story card:** [ch5731](https://app.shortcut.com/simpledotorg/epic/5731), [ch6103](https://app.shortcut.com/simpledotorg/story/6103/rename-the-monthly-district-data-download-in-district-reports-to-be-monthly-facility-data)

## Because

CVHOs can already download a CSV containing the essentials of a monthly report for a district, broken down by facilities. They want this at the state level, broken down by districts.

## This addresses

This clones the district report into a state report and cleans up the code a bit, using only the `Reports::Repository` now.

This also renames:
- `Monthly state data` --> `Monthly district data`
- `Monthly district data` --> `Monthly facility data`

Note that this is purely a clone for now. A future PR will merge this code with the district report so we only have one export service.

## Test instructions

1. Go to the test app
2. Click on "Reports", make sure you're looking at the highest level in the hierarchy (a State)
3. Click the "Download" button top right
4. Click "Monthly district data"
